api/app/Actions/Auth/sendLinkForgotPasswordAction.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Actions\Auth;
+
+use App\Exceptions\Auth\UserWithThisEmailDoesNotExist;
+use App\Repositories\User\UserRepository;
+use Illuminate\Auth\Notifications\ResetPassword;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Support\Facades\Lang;
+use Illuminate\Support\Facades\Password;
+
+final class sendLinkForgotPasswordAction
+{
+    private UserRepository $userRepository;
+
+    public function __construct(UserRepository $userRepository)
+    {
+        $this->userRepository = $userRepository;
+    }
+    public function execute(sendLinkForgotPasswordRequest $request)
+    {
+        if (!$this->userRepository->getByEmail($request->getEmail())) {
+            throw new UserWithThisEmailDoesNotExist($request->getEmail());
+        }
+        ResetPassword::toMailUsing(function ($notifiable, $token) {
+            $linkReset = url(config('app.url') . route('password.reset', ['token' => $token, 'email' => $notifiable->getEmailForPasswordReset()], false));
+            $count = config('auth.passwords.' . config('auth.defaults.passwords') . '.expire');
+            return (new MailMessage())
+                ->subject(Lang::get('Reset Password from Schedulia'))
+                ->view('emails.sendLinkResetPassword', ['linkReset' =>$linkReset, 'count'=>$count]);
+        });
+        $credentials = ['email' =>$request->getEmail()];
+        Password::sendResetLink($credentials);
```

Comment:
I am not 100% sure, but seems that `ResetPassword::toMailUsing(...)` and `Password::sendResetLink($credentials);` does the same thing. The second method does the thing you need (you can check `vendor/laravel/framework/src/Illuminate/Auth/Passwords/PasswordBroker.php` to see it's actions).

Reply:
ResetPassword::toMailUsing(...) add call back with help of which we can make customization of our letter to user.